app/models/wpcc/minimum_contribution_calculator.rb

```diff
@@ -7,11 +7,15 @@ def eligible_salary
   end
 
   def employee_percent
-    1
+    percentage(:employee)
   end
 
   def employer_percent
-    1
+    percentage(:employer)
+  end
+
+  def below_threshold
+    false
```

Comment:
I will be updating this method in the next PR/[TP Card](https://moneyadviceservice.tpondemand.com/entity/8905-wpcc-update-default-contributions-for-employees)  as this calculator is going to include logic in here